website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,212 @@
+
+The following will provide a step-by-step guide on setting up DuckDB for use in both local environments and web browsers.
+
+For more information on the setup of DuckDB, you can refer to [DuckDBsetup](/docs/core/connect-data-platform/duckdb-setup).
+
+
+<Tabs>
+
+<TabItem value="local" label="Local">
+
+1. First, you'll need to clone this repository.
+
+    For a more comprehensive understanding and additional information, refer to the [Git Guide](https://github.com/git-guides/git-clone).
+
+    You can access the repository for DuckDB by following the URL:
+
+    ```bash
+    git clone https://github.com.dbt-labs/jaffle_shop_duckdb.git
+
+    ```
+
+2. Change into the docs-duckdb directory from the command line:
+
+    ```Jinja
+
+    cd jaffle_shop_duck_db
+
+    ```
+
+
+3. Install dbt and DuckDB in a virtual environment.
+
+    <Expandable alt_header="Example for MAC" >
+
+    ```Jinja
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install --upgrade pip
+    python3 -m pip install -r requirements.txt
+    source venv/bin/activate
+
+    ```
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows" >
+
+    ```Jinja
+
+    python -m venv venv
+    venv\Scripts\activate.bat
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\activate.bat
+
+    ```
+
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows PowerShell" >
+
+    ```Jinja
+
+    python -m venv venv
+    venv\Scripts\Activate.ps1
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\Activate.ps1
+
+    ```
+    </Expandable>
+
+
+4. Ensure your profile is setup correctly from the command line by running the following:
+
+
+    - [dbt compile](https://docs.getdbt.com/reference/commands/compile) &mdash; generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) &mdash; compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) &mdash; compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) &mdash; compiles, runs, and tests your project
+    - [dbt docs generate](/reference/commands/cmd-docs#dbt-docs-generate) &mdash; generates your project's documentation.
+    - [dbt docs serve](/reference/commands/cmd-docs#dbt-docs-serve) &mdash; starts a webserver on port 8080 to serve your documentation locally and opens the documentation site in your default browser.
+
+    For complete details, refer to the [dbt command reference](/reference/dbt-commands).
+
+:::note
+
+If you decide to run this project in your own data warehouse (outside of this DuckDB demo) and steps fail, it might mean that you need to make small changes to the SQL in the models folder to adjust for the flavor of SQL of your target database. Definitely consider this if you are using a community-contributed adapter.
```

Comment:
It will most certainly fail elsewhere - they'll have to reconfigure their project files
```suggestion
The steps will fail if you decide to run this project in your data warehouse (outside of this DuckDB demo). You will need to reconfigure the project files for your warehouse. Definitely consider this if you are using a community-contributed adapter.
```